projects/kubernetes/kubernetes/1-18/patches/0004-EKS-PATCH-volume-plugin-requests-patch.patch

```diff
@@ -5,14 +5,18 @@ Subject: [PATCH 04/16] --EKS-PATCH-- volume plugin requests patch
 
 Mitigation for CVE-2020-8555
 
+To maintain backwards compatibility, the "EXTRA_PROXY_BLACKLIST_CIDR" env var must be accepted, despite containing the
+non-inclusive term "BLACKLIST". The "EXTRA_PROXY_DENYLIST_CIDR" env var has the same functionality and should be used
+instead of it. Future releases should consider supporting only the inclusive variable.
+
 Signed-off-by: Jackson West <jgw@amazon.com>
 ---
- pkg/proxy/util/utils.go                       | 59 ++++++++++++++++++-
+ pkg/proxy/util/utils.go                       | 63 +++++++++++++++++++-
  .../heketi/client/api/go-client/client.go     | 19 +++++-
  vendor/github.com/quobyte/api/quobyte.go      | 18 +++++-
  vendor/github.com/storageos/go-api/client.go  |  5 +-
- .../github.com/thecodeteam/goscaleio/api.go   | 11 +++-
- 5 files changed, 106 insertions(+), 6 deletions(-)
+ .../github.com/thecodeteam/goscaleio/api.go   |  9 +++
```

Comment:
This change may not be clear the following context. This patch is edited from how it was generated. There were two changes that we manually deleted from the patch file. However, we didn't manually update the change count to reflect this. This revision corrects this issue.